static-assets/components/cstudio-forms/controls/numeric-input.js

```diff
@@ -106,11 +106,12 @@ YAHOO.extend(CStudioForms.Controls.numericInput, CStudioForms.CStudioFormField,
 	 * @param el element
 	 */
 	count: function(evt, countEl, el) {
-		var max = this.getAttribute("max") ,
-			min = this.getAttribute("min");
-		if(max && parseInt(this.value) > max){
+	  var el = (el) ? el : this,
+        max = this.getAttribute("max") ,
+			  min = this.getAttribute("min");
```

Comment:
Where is getAttribute declared?

Reply:
@rart yeah sorry, I need https://github.com/craftercms/studio-ui/pull/1475 to be merged so I can update this branch